### ISSUE TITLE
some problems with emojis (Espeak ng for Android)

### ISSUE THREAD

**Issue by errotech** [espeak-ng/espeak-ng#1567]
When using eSpeak with Talkback, emojis aren't or just partialy spoken.

**Comment by MH140000** [comment]
Yes. It is the same for me. If there is a space in the emojis, only the first word is read

**Comment by kirill-jjj** [comment]
@errotech, @MH140000, this isn't only on android, disable Include Unicode Consortium data (including emoji) when processing characters and symbols in nvda, and you will get the same. Noone want's to fix it, partly because screen reader have built-in support for emoji on computer's, and partly, as I understand, it is very hard to fix that

**Comment by errotech** [comment]
Yes, you're right. But the weird thing is that there are emojis which eSpeak reads correctly. For example ❤️ = "red", 😂 = "face with tears of joy"

**Comment by hozosch** [comment]
Well, red heart isn't read correctly, since the heart is missing. But if the second one is correctly, that is indeed strange.

**Comment by alex19EP** [comment]
Retested on current master: the specific problems reported here are fixed — ❤️ reads "red heart" (not just "red"), and multi-word emoji descriptions are read in full even for consecutive emoji. As noted in the comments this was an engine issue, not Android-specific; the fix will reach the Android app with the next engine update.

The remaining limitation — ZWJ composite emoji (👩‍🔬 etc.) being read as their components — is tracked in #308.

Closing; please reopen if simple emoji are still misread with a current build.
